### DISCUSSION TITLE
Prerender error with Static Paths with svg file

### DISCUSSION THREAD

**Discussion by tcolinpa** [vercel/next.js#19379] [Help]
## Describe the bug

I've got a page with dynamic routing such as [slug].tsx and it uses getStaticProps to fetch data from a headless CMS (Prismic) and getStaticPaths.

```
export const getStaticPaths: GetStaticPaths = async () => {
  const posts = await client().query([
    Prismic.Predicates.at('document.type', 'post')
  ])

  const paths = posts.results.map(post => {
    return {
      params: { postSlug: post.uid }
    }
  })

  return {
    paths,
    fallback: true
  }
}

export const getStaticProps: GetStaticProps<PostProps> = async context => {
  const { postSlug } = context.params

  var post = await client().getByUID('post', String(postSlug), {})
  var category = await client().getByUID(
    'category',
    String(post.data.category.slug),
    {}
  )

  if (!post) {
    post = null
  }

  return {
    props: {
      post: post,
      category: category
    },
    revalidate: 60
  }
}
```

This page imports my Header component which contains a .svg file named Logo (using inline-react-svg -> import Logo from 
'logo.svg'). When using `yarn dev` it works, but when using `yarn build`, I get the following error:

```
[=   ] info  - Generating static pages (10/14)
Error occurred prerendering page "/how-to-operate-during-coronavirus". Read more: https://err.sh/next.js/prerender-error
TypeError: undefined is not a function
    at Logo (/Users/tiagocolin/Projects/secret/code/blog/.next/server/pages/[postSlug].js:737:29)
```

I found two workarounds:
1. deleting the svg file from the Header component
2. using the svg file and using the following getStaticPaths 

```
export const getStaticPaths: GetStaticPaths = async () => {
  return {
    paths: [],
    fallback: true
  }
}
```

What am I missing here? Why using the first getStaticPaths and importing the svg file in the Header component doesn't work at build?

## System information

- OS: macOS
- Browser: Chrome
- Version of Next.js: v10.0.1
- Version of Node.js: v10.16.3

**Comment by mfellner** [comment]
I don't know the root cause of the error but I was able to simply use [SVGR](https://medium.com/frontend-digest/how-to-import-svgs-into-nextjs-8ec6100e613f#9201) instead of inline-react-svg.
